src/engines/html5/html5.js

```diff
@@ -507,10 +509,6 @@ export default class Html5 extends FakeEventTarget implements IEngine {
             )
           );
         });
-      } else if (typeof this._el.webkitSetPresentationMode === 'function') {
-        this._el.webkitSetPresentationMode('picture-in-picture');
-        // Safari does not fire this event but Chrome does, normalizing the behaviour
-        setTimeout(() => this.dispatchEvent(new FakeEvent(Html5EventType.ENTER_PICTURE_IN_PICTURE)), 0);
```

Comment:
don't you miss this event on your change?

Reply:
Didn't it change on https://github.com/kaltura/playkit-js/pull/483?

Reply:
no. here I changed `enterPictureInPicture`. there I changed `exitPictureInPicture`.